### PR TITLE
Update security-assessment.md

### PR DESCRIPTION
**Pull request by szymonkowalczykzf**
Added Mermaid Diagram.
Additionally removed Vulnerabilities number 5&7 (Rate Limitting & encryption of data at rest) due to the fact that they are still open and I believe we had an agreement to upload here only the addressed vulnerabilities.

I have already accepted the threat modeling requirement for the Quality Gate 24 March 2024.

Please review and approve.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs/security-assessment.md [hunk 1]
```diff
@@ -39,7 +40,84 @@ Currently the PURIS Application have 1 main functionality:
...
+
+    D-->|Data Read & Write \n TCP9092|D5
+    D3-->|Data Read & Write \n TCP9092|D5
+   
+    D4-->|Data Read & Write \n TCP9092|D5
+
+    D3-->|Forwarding Data Response \n HTTPS Protocol|B
+
+    B-->|Forwarding Data Requests \n HTTPS Protocol|D3
+    B-->|List catalog negotiations \n and transfers, \n Read Access \n HTTPS Protocol|D2
```
Comment (tom-rm-meyer-ISST) [inline docs/security-assessment.md:79]: please adjust to "List catalog, negotiations \n and transfers, ..." :)

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Please add one comma for readability. Rest ist top! Wasn't aware of the native integration of mermaid in GIthub! :)

**Comment by szymonkowalczykzf** [comment]
@tom-rm-meyer-ISST Change implemented, did not catch up this one earlier, thanks.

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM, thanks for providing!

**Pull request merged**